### ISSUE TITLE
Authorization Failed

### ISSUE THREAD

**Issue by Cowseer** [Healix/Gw2Launcher#91]
Hey, I have an authenticator setup to allow the current network to be authorized and remembered, however it says "Authorization Failed" whenever the 6 digit code is entered.

Thank you for your time.

**Comment by Cowseer** [comment]
Update: Even though it says "Authorization Failed", the network gets verified/remembered. I have to uncheck "Enable authorization for this account" to launch it or I could disable "Cancel launching when authorization is not successful" as a workaround.

**Comment by Healix** [comment]
Disable network authentication under Settings > Security. ArenaNet added protections against automating account management and I didn't want to risk bypassing it, so this feature is no longer supported.

In the meantime, for an easier time entering the code into the launcher, you can either copy/paste it (Settings > Style > configure middle click to show/copy) or simply hold right click on the launcher to paste it (enabled under Settings > Tools).

**Comment by Cowseer** [comment]
Thank you, that's exactly what I did after playing around with it.

**Comment by pandawelch** [comment]
Is it possible that if 
1. Account is not new (i.e. Eula popup)
2. Automatic Logins is turned on
3. The login window exists longer than expected after inputting the account credentials
4. Authenticator is added
5. (Optional) Checked against known working IPs forthe account

The authenticator code, checkbox and 'Enter' can be inputted? About here: https://github.com/Healix/Gw2Launcher/blob/a8bcbadb52d5b50ee073e97a734f39c92c6d13b5/Gw2Launcher/Client/Launcher_Autologin.cs#L545

**Comment by Healix** [comment]
I have a few potential solutions for automating the code, I just haven't gotten around to testing. At this point, I'm waiting for EoD in case they finally upgrade the launcher like China did. Worst case, the simplest option is just blindly pasting the code every few seconds, and if something accepts the paste, I assume it was entered.

**Comment by nakami** [comment]
any update on this?

**Comment by Healix** [comment]
> any update on this?

No. I've basically had no time this year (problems keep happening).
